scripts/tests/scripts_regression_tests.py

```diff
@@ -253,7 +253,7 @@ def test_a_createnewcase(self):
             shutil.rmtree(testdir)
 
         cls._testdirs.append(testdir)
-        run_cmd_assert_result(self, "%s/create_newcase --case %s --compset X --res f19_g16 --output-root %s" % (SCRIPT_DIR, testdir, cls._testroot), from_dir=SCRIPT_DIR)
+        run_cmd_assert_result(self, "%s/create_newcase --case CreateNewcaseTest --script-root %s --compset X --res f19_g16 --output-root %s" % (SCRIPT_DIR, testdir, cls._testroot), from_dir=SCRIPT_DIR)
```

Comment:
I'll do that later today. Would you want me to emulate the structure that the coupled team wants, or is just leaving it the way it is fine?